wsd/ClientSession.cpp

```diff
@@ -1120,7 +1120,7 @@ bool ClientSession::_handleInput(const char *buffer, int length)
         if (tokens.equals(0, "key"))
             _keyEvents++;
 
-        if (isEditable() && COOLProtocol::tokenIndicatesDocumentModification(tokens))
+        if (isEditable() && COOLProtocol::tokenIndicatesDocumentModification(tokens)) // check this function: tokenIndicatesDocumentModification
```

Comment:
forgotten todo? what is that comment about?

Reply:
Yep, let me check why i did it, i will probably remove that comment :)